framework/src/auxkernels/TagVectorAux.C

```diff
@@ -15,21 +15,28 @@ InputParameters
 TagVectorAux::validParams()
 {
   InputParameters params = TagAuxBase<AuxKernel>::validParams();
-  params.addRequiredParam<TagName>("vector_tag", "Tag Name this Aux works on");
-  params.addClassDescription("Couple a tag vector, and return its nodal value");
+  params.addClassDescription("Couple a tagged vector, and return its DOF value");
+  params.addRequiredParam<TagName>("vector_tag", "Name of the vector tag");
+  params.addParam<bool>("remove_variable_scaling", false, "Remove variable scaling from DOF value");
   return params;
 }
 
 TagVectorAux::TagVectorAux(const InputParameters & parameters)
   : TagAuxBase<AuxKernel>(parameters),
+    _unscaled(isParamSetByUser("scaled") ? !getParam<bool>("scaled")
```

Comment:
_unscaled = "scaled"
What could possibly go wrong? 🫤 It might be worth adding a comment to make it clear that this is intentional.

Reply:
unscaled = !scaled here
it s fine

Reply:
Aha, now I see what @bwspenc meant. You missed that `!` operator LOL.